Formula/lib/libevdev.rb

```diff
@@ -21,33 +21,17 @@ def install
 
   test do
     (testpath/"test.c").write <<~EOS
```

Comment:
```suggestion
    (testpath/"test.c").write <<~C
```
Let's use language-specific heredocs delimiters